video_xblock/backends/html5.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Html5 Video player plugin."""
+
+import json
+import re
+
+from video_xblock import BaseVideoPlayer
+
+
+class Html5Player(BaseVideoPlayer):
+    """
+    Html5Player is used for videos by providing direct URL.
+    """
+
+    url_re = re.compile(r'^(https?|ftp)://[^\s/$.?#].[^\s]*.(mpeg|mp4|ogg|webm)')
+
+    metadata_fields = []
```

Comment:
Make `BaseVideoPlayer.metadata_fields` non-abstract and drop this line and in other backends.